features/pull_request.feature

```diff
@@ -812,3 +812,36 @@ BODY
       Error creating pull request: Temporary Redirect (HTTP 307)
       Refused to follow redirect to https://disney.com/mouse\n
       """
+
+  Scenario: Default message with --push
+    Given the text editor adds:
```

Comment:
Done. I do want to run the editor here to ensure that the automatically-generated message is correct.